guest/toolbox/example_test.go

```diff
@@ -18,18 +18,24 @@ package toolbox_test
 
 import (
 	"context"
+	"fmt"
 	"os"
 	"os/exec"
 
 	"github.com/vmware/govmomi/find"
 	"github.com/vmware/govmomi/guest/toolbox"
 	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/test"
 	"github.com/vmware/govmomi/vim25"
 	"github.com/vmware/govmomi/vim25/types"
 )
 
 func ExampleClient_Run() {
 	simulator.Run(func(ctx context.Context, c *vim25.Client) error {
+		if !test.HasDocker() {
+			fmt.Println("Linux")
```

Comment:
was this print left intentionally here?

Reply:
Yes, it was already the expected output of this Example test, see `// Output:` below.
But in Example tests, we can't use `testing.T.Skip()`, so just print the expected output and return.

Reply:
aaaaah indeed sorry! missed the output assertion